interpreter/code_interpreter.py

```diff
@@ -117,6 +117,15 @@ def start_process(self):
     threading.Thread(target=self.save_and_display_stream,
                      args=(self.proc.stderr, True), # Passes True to is_error_stream
                      daemon=True).start()
+    
+  def stop(self):
+    """
+    Stops the interpreter process.
+    """
+    if self.proc:
+      self.proc.terminate()
+      self.proc.stdin.close()
+      self.proc.stdout.close()
```

Comment:
I think these changes probably need to move to [`interpreter/core/core.py`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/core/core.py) now that the code has been refactored.

Reply:
Of course, I'll come back to modify this commit.

Reply:
What do you think about turning off stdin and stdout directly in the terminate method?